src/configuration.ts

```diff
@@ -44,7 +44,7 @@ export interface IConfigurationFile {
      * Other linter options, currently for testing. Not publicly supported.
      */
     linterOptions?: {
-        typeCheck?: boolean;
+        typeCheck?: boolean,
```

Comment:
So a property in an interface should be treated like a type alias?
What about a property in a class, or a return type of a method?